custom/icds/views/custom_sms_report.py

```diff
@@ -47,3 +59,22 @@ def post(self, request, *args, **kwargs):
                 for message in message_list:
                     messages.error(self.request, message)
         return self.get(*args, **kwargs)
+
+
+def _prepare_display_message(reports_in_progress, report_count):
+    message = _('Currently {reports_count} {report_text} for duration\n').format(
+        reports_count=report_count,
+        report_text='reports' if report_count>1 else 'report',
+    )
+    for index, report in enumerate(reports_in_progress):
+        message+=_('{start_date}-{end_date}').format(
+            start_date=report['start_date'],
+            end_date=report['end_date']
+        )
+        if index!=report_count-1:
+            message+=', '
+    message+=_(' {is_or_are} in progress').format(
+        is_or_are='are' if report_count>1 else 'is',
+    )
+    return message
```

Comment:
I think its worth keeping it plain simple.
```
Reports in progress
2020-01-02 - 2020-02-02
```
Also, if you choose to just save the dates as strings as suggested earlier, you can use it directly here.